services/payments/src/simcore_service_payments/db/payments_transactions_repo.py

```diff
@@ -1,9 +1,190 @@
-class PaymentsTransactionsRepo:
-    #
-    # Next PRs should move most of the implementations in
-    # services/web/server/src/simcore_service_webserver/payments/_db.py
-    # here.
-    # The transition should put all the implementations in the simcore_postgres_database first
-    # so it is usable temporarily in both the webserver and here
-    #
+import datetime
+from decimal import Decimal
+
+import sqlalchemy as sa
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from pydantic import HttpUrl, PositiveInt, parse_obj_as
+from pydantic.errors import PydanticErrorMixin
+from simcore_postgres_database.models.payments_transactions import (
+    PaymentTransactionState,
+    payments_transactions,
+)
+from sqlalchemy.ext.asyncio import AsyncEngine
+
+from ..models.db import PaymentsTransactionsDB
+from ..models.payments_gateway import PaymentID
+
+#
```

Comment:
I am really not sure these comments bring anything...
The naming is already quite clear.. why bother?

Reply:
sanity :-)
I use this to separate sections so later I can move them to other files (when there are many or need to be shared)

Reply:
which you did not do at this point... just saying that I tend to erase that kind of comments... especially when I find them at the beginning of the files when isort moved them and nobody saw it. but that is your call.